lib/remote-template.js

```diff
@@ -108,6 +119,10 @@ async function getRemoteElmJson(
   return TemplateDependencies.update(options, elmJson);
 }
 
+/**
+ * @param {Template} template
+ * @param {string} commit
+ */
```

Comment:
Should this not a return type annotation? (Same of other functions in this file)?

Reply:
They should, but I used the IDE inferer to get started, which skips them, unfortunately, and I didn't realize it until I was done.
TS infers them though, so I think it can be left for a follow-up.